HomeWork11/task2.py

```diff
@@ -0,0 +1,4 @@
+class MyCustomException(Exception):
+    def __init__(self, message="Custom exception is occurred"):
+        self.message = message
+        super().__init__(self.message)
```

Comment:
nit: message could be set directly into __init__ method
```suggestion
        super().__init__(message)
```